src/fe-small-advertisements/src/App.tsx

```diff
@@ -1,11 +0,0 @@
-import React from 'react';
-
-function App() {
-  return (
-    <>
-      <h1>Hello World</h1>
-    </>
-  );
-}
-
-export default App;
```

Comment:
Lets leave the app where it used to be. Later in this file we will be doing our routing. Or if you insist moving it. Make a folder for it and also move the `App.test.tsx` in this folder. Now the app and the tests of it are in different places.

Reply:
I moved it back. I really have to read about folder structure, I already started to read about DDD and I was reading about `features-folder` because that is the way the official docs of `redux-toolkit` recommend.

> Note that while it doesn't matter how you actually structure your folders and files, we've found that a "feature folder" approach usually works better for most applications. The file name is also entirely up to you, but a convention of someFeatureSlice.js is reasonable to use.

For this commit  I created a /features/person folder and put the personSlice.ts there. I just to put it somewhere until I read more about DDD, is that fine? Or is that sloppy of me?

 Sometimes I am not sure what to do in cases like this one: On the one hand, you asked me to build just a showcase and I have to decide how to make that showcase the best way, but I also don't have in my mind the whole picture of the folder structure. So it is hard to decide where to put what.

Reply:
right. you right this wasnt clear. but we discuss the file structure we want the project to have in general. Now if I say this is alright and we merge this means also we will continue with this structure as it is now. 

So from my point of view I will have to bring this on your attention. Now you right that for this case it doesnt really matter. We can leave it like this however you will have to change it a little later. So up to you what you prefer. 

I prefer to merge on `main` only code that is production level.